server/views/referrals/riskInformation.njk

```diff
@@ -28,6 +29,25 @@
 
       {{ govukTable(roshAnalysisTableArgs(govukTag)) }}
 
+      {{ govukDetails({
+          summaryText: "Definitions of risk levels",
+          html:
+            '<ul class="govuk-list govuk-list--bullet">
```

Comment:
Not a blocker, but we tend to construct this sort of thing in the `view` layer as `govUkDetailsArgs` and pass it to the macro to keep the template a bit smaller.

Reply:
👍